### ISSUE TITLE
Track the number of skills retrieved by the user

### ISSUE THREAD

**Issue by Randomacy** [Poapedu/poapedu-app#13]
Same thing as poapedus - just need to retrieve the information from DB instead of 0

**Comment by harishkotra** [comment]
@Randomacy please confirm if this is the flow:

1. User retrieves their certificate
2. User Mints and NFT and then the skills are saved to the database.
3. Skills stored for the minted NFTs are to be retrieved in their profile page. Is that right?

If that's the case, we don't nee d

- [ ] Remove the Add Skill functionality from the `EditProfile.vue` page.
- [ ] Fetch skills (if available) for the logged in user and show them as labels under their profile information in `EditProfile.vue`

**Comment by Randomacy** [comment]
Yes correct. We can show the skills as labels, though I am concerned that it might end up being too bulky if the user has too many skills
